ingest/v2.md

```diff
@@ -0,0 +1,101 @@
+
+Here's an example in cURL:
+
+```bash
+curl -X "POST" "https://nom.telemetrydeck.com/v2/" \
+     -H 'Content-Type: application/json; charset=utf-8' \
+     -d $'[
+        {
+          "appID": "AAAA-BBBBBBBB-CCCC-DDDD",
+          "clientUser": "myClientUserHash",
+          "type": "Actions.LoginView.loginButtonSucceeded"
+        }
+      ]'
+```
+
+## Body Structure
+
+The post body should be an **array of JSON objects**, because you can send multiple signals at once. Each signal object **must** have these properties:
+
+| Property     | Type   | Description                                                                                                                                                                                                                              |
+| ------------ | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `appID`      | string | Your app's ID                                                                                                                                                                                                                            |
+| `clientUser` | string | A hash of the user's ID. This should always be the same for the same user.                                                                                                                                                               |
+| `type`       | string | The type of signal. While it is not enforced, we recommend structuring your signal names in namespaces separated by dots, with the signal type beginning with a lower case letter and any namespaced beginning with an uppercase letter. |
+
+The following properties are optional:
+
+| Property     | Type   | Description                                                                                 |
+| ------------ | ------ | ------------------------------------------------------------------------------------------- |
+| `sessionID`  | string | The user's session ID. This should be the same value for the same session/user combination. |
+| `payload`    | object | A JSON object with additional data.                                                         |
+| `isTestMode` | bool   | If `true`, the signal will be excluded from production queries. Defaults to `false`.        |
+
+## Payload
+
+The `payload` object can contain data you want to send to TelemetryDeck. It must be a **JSON object containing primitives**. Here's an example:
+
+```json
+{
+  "appID": "AAAA-BBBBBBBB-CCCC-DDDD",
+  "clientUser": "myClientUserHash",
+  "type": "Curl.Development.apiCall",
+  "sessionID": "mySessionID",
+  "isTestMode": false,
+  "payload": {
```

Comment:
Small detail, but in the table above these two have a different order than here. Definitely not a blocker, but consistency could help with clarity.

Reply:
Good catch, I updated the order in the table